### DISCUSSION TITLE
This is an arrow

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#564] [Announcements]
This is an arrow!
It might not look like it.
Because it's a weekly DreamBerd update.

This week, there's a new video - made by **Berd** of Dream**Berd**.

[Here it is!](https://www.youtube.com/watch?v=DNBKdU6XrLY)

Thank you for reading this week's release notes.

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/DreamBerd/releases/tag/varrow'>This is an arrow</a>.</em>

**Comment by skifli** [comment]
That arrow was straight to the point.

**Comment by MaddyGuthridge** [comment]
That video is seriously a work of art - it was so cool to watch!

**Comment by loglot** [comment]
where will this project go, now that berd left

**Comment by D-u-c-k-s-e-l** [comment]
that was fun
